### PR TITLE
Unlink card expenses docs

### PR DESCRIPTION
**Pull request by Code0987**

### CODE REVIEW

File: reference/admin.yaml [hunk 1]
```diff
@@ -2112,6 +2112,13 @@ components:
           type: string
           nullable: true
           example: '2020-07-20'
+    is_implicit_merge_blocked:
+      type: boolean
+      nullable: true
+      description: >
+        This indicates if the expense can be considered for implicit merge or
+        not
```
Comment (Kulsekar4) [inline reference/admin.yaml:2120]: just check if this looks good in preview @Code0987 sir.

File: src/components/schemas/fields.yaml [hunk 2]
```diff
@@ -691,3 +691,11 @@ bank_ifsc:
   description: >
       IFSC code of the bank account of the Employee
   pattern: '^[A-Z]{4}0[A-Z0-9]{6}$'
+
+
+is_implicit_merge_blocked:
+  type: boolean
+  nullable: true
+  description: |
+    This indicates if the expense can be considered for implicit merge or not
```
Comment (Kulsekar4) [inline src/components/schemas/fields.yaml:700]: `This indicates if the expense can be merged implicitly or not`
^ My suggestion, if it doesn't break the context then can add this.

File: reference/admin.yaml (hunk 1 above)
Comment (Code0987) [reply reference/admin.yaml:2120]: Ohh, it's the bot messing up :/ But it looks fine
Screenshot 2022-05-23 at 5 29 09 PM

File: src/components/schemas/fields.yaml (hunk 2 above)
Comment (Code0987) [reply src/components/schemas/fields.yaml]: Done

**Review by Kulsekar4 — APPROVED** [review]

**Review by modernwarfareuplink — APPROVED** [review]

**Pull request closed without merging**